cmd/main.go

```diff
@@ -118,17 +119,25 @@ func parseConfig() (Config, error) {
 			"Command-line flags override configuration from this file.")
 	flag.StringVar(&spireAPISocketFlag, "spire-api-socket", "", "The path to the SPIRE API socket (deprecated; use the config file)")
 	flag.BoolVar(&expandEnvFlag, "expand-env", false, "Expand environment variables in SPIRE Controller Manager config file")
+	opts := zap.Options{
+		Development: true,
```

Comment:
Why use development mode by default? Production mode uses info by default (and JSON) whereas development mode uses debug and console (human readable).